app/graphql/mutations/create_grading.rb

```diff
@@ -257,7 +257,21 @@ def send_feedback
           timeline_event: submission
         )
 
-      StartupFeedbackModule::EmailService.new(startup_feedback).send
+      grading_details = submission.evaluation_criteria.map.with_index { |criteria, index|
```

Comment:
This is not the right place to have the computation of grading details. This is something that you are computing specifically for displaying information in the mailer. So this computation can be moved to the `EmailService`

Another reason is that all of this is done inline during the request, which is not required. Email itself is a deferred job and this can definitely be pushed to the mailer considering it is very specific for this purpose.